[DEVX-1067] Add docs for playwright project (#936)
saucelabs/sauce-docs: docs/testrunner-toolkit/configuration/playwright.md

```
---

### `configFile`
<p><small>| OPTIONAL | STRING |</small></p>

> The designated `playwright` configuration file. `saucectl` determines related files based on the location of the config file. Supports both TypeScript and JavaScript files.
> If it's not set, `saucectl` will pick default playwright config file(`playwright.config.ts` or `playwright.config.js`).

Reviewer: ```suggestion
The path (relative to `rootDir`) to your Playwright configuration file. `saucectl` determines related files based on the location of this config file. Supports both TypeScript and JavaScript files.
If it's not set, `saucectl` defaults to `playwright.config.ts` or `playwright.config.js`.
```

```yaml

>   configFile: config.ts

Author: It should be a relative path and under `rootDir`

      browserName: "firefox"
      headful: false
      slowMo: 1000
      project: "project name"
```

__Description__: This field is for specific test run parameters, for example:
* `browserName` ( *string* ) : the browser in which to run tests
* `headful` ( *boolean* ) : whether to run browsers in headless mode
* `sloMo` ( *int* ) : whether to implement artificially slow load times in milliseconds

> * `project` ( *string* ): the playwright project name

Author: sure.
